src/phpDocumentor/Descriptor/Builder/PhpParser/FileAssembler.php

```diff
@@ -267,7 +263,7 @@ public function leaveNode(Node $node)
                 $this->createDescriptorFromNodeAndAddToCollection($node, $this->fileDescriptor->getInterfaces());
                 break;
             case 'PhpParser\Node\Stmt\Function_':
-                $this->createDescriptorFromNodeAndAddToCollection($node, $this->fileDescriptor->getFunctions());
```

Comment:
I will have to rewrite the Assemblers in to factory strategies, So this will be removed in a next pr.